src/Compilers/Core/Portable/CodeGen/PrivateImplementationDetails.cs

```diff
@@ -307,7 +333,7 @@ internal MappedField(string name, Cci.INamedTypeDefinition containingType, Cci.I
 
         public Cci.IMetadataConstant GetCompileTimeValue(EmitContext context) => null;
 
-        public ImmutableArray<byte> MappedData => _block;
+        public abstract ImmutableArray<byte> MappedData { get; }
```

Comment:
Have callers been updated to handle a default `ImmutableArray`?  It looks like that used to be impossible.